.github/workflows/tests.yml

```diff
@@ -19,6 +19,14 @@ jobs:
       - run: scripts-dev/generate_sample_config.sh --check
       - run: scripts-dev/config-lint.sh
 
+  check-schema-delta:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: "pip install 'click==8.1.1' 'GitPython>=3.1.20'"
```

Comment:
Ideally we'd be using the poetry locked environment here (and indeed in `check-sampleconfig` and `lint-newsfile`)---but I won't block this on that.

Reply:
I mainly did this as its so much faster than installing all the deps.

Reply:
True, but FWIW there's a bunch of caching in https://github.com/matrix-org/setup-python-poetry which should help mitigate that